specs/mocks/geolevels.json

```diff
@@ -13,29 +13,29 @@
 }, {
     "id": "fr/canton",
     "name": "French canton",
-    "parents": ["fr/county"]
+    "parents": ["fr/departement"]
 }, {
-    "id": "fr/county",
+    "id": "fr/departement",
     "name": "French county",
     "parents": ["fr/region"]
 }, {
     "id": "fr/district",
```

Comment:
You're right, it's a mock. If we don't rename other territories yet, just keep it like that.